packages/website/ts/components/ui/metamask_mascot.tsx

```diff
@@ -0,0 +1,61 @@
+import * as debounce from 'debounce';
```

Comment:
Can probably uninstall this?
https://lodash.com/docs/4.17.10#debounce

Reply:
Oops i accidentally pushed my metamask animation stuff with this branch